packages/select/src/vaadin-select.js

```diff
@@ -533,34 +569,48 @@ class Select extends DelegateFocusMixin(FieldMixin(SlotMixin(ElementMixin(Themab
 
     const label = selected.getAttribute('label');
     if (label) {
-      labelItem = this.__createItem(label);
+      labelItem = this.__createItemElement({ label, component: 'vaadin-item' });
```

Comment:
Should this also create a `<vaadin-select-item>`?

Reply:
I remember we earlier discussed it with @web-padawan and, for some reason, decided to use `vaadin-item` here. Now I'm sure that it was a totally wrong decision, especially in the light of using the `label` attribute. Thank you for the good catch. Fixed.